app/controllers/api/v1/sauces_controller.rb

```diff
@@ -1,5 +1,5 @@
 class Api::V1::SaucesController < ApiController
   def show    
-    render json: Sauce.find(params[:id])
+    render json: Sauce.find(params[:id]), include: "reviews,reviews.votes"
```

Comment:
add a space after your comma

Reply:
Ill admit, Im less wild about this, as its an alternative to our established serializer pattern. Ill allow it if you really want since its not wrong, but encourage handling in the sauce serializer

Reply:
This format is required for the string "reviews,reviews.votes", as adding a comma will break this "include:" statement (see "Included" section of documentation below).

https://github.com/rails-api/active_model_serializers/blob/a032201a91cbca407211bca0392ba881eef1f7ba/docs/general/adapters.md#included